Tests/RestaurantsPage_test.py

```diff
@@ -12,3 +13,25 @@ def test_title(self, restaurant):
         title = self.restaurant.get_title()
 
         assert title == restaurant['title']
+
+    def test_logo_visible(self, restaurant):
+        self.restaurant = RestaurantsPage(self.driver, restaurant['url'])
+
+        assert BasePage.is_visible(RestaurantsPage.LOGO) == True
```

Comment:
no need to use '==' operand, as is_visible already return boolean value. If assert True - test pass, otherwise - failed.

[ assert BasePage.is_visible(RestaurantsPage.LOGO) ]  provides the same result as [ assert BasePage.is_visible(RestaurantsPage.LOGO) == True ]